### PR TITLE
Remove extraneous `\`

### LINKED ISSUE #11259: Issue with "Accessibility and Spacial Patterns": Escaped asterisks on page
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Accessibility_and_Spacial_Patterns

#### What information was incorrect, unhelpful, or incomplete?

The asterisks for the section containing the definition of Spatial Localization are escaped, preventing proper italicization.

#### Specific section or headline?

Top of page

#### What did you expect to see?

Italicized text

#### Did you test this? If so, how?

N/A

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/accessibility_and_spacial_patterns`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Accessibility_and_Spacial_Patterns
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/accessibility_and_spacial_patterns/index.md
* Last commit: https://github.com/mdn/content/commit/697f76468368a35a485a608ec70aff2278e63f5b
* Document last modified: 2021-10-13T21:04:39.000Z

</details>

### PR DESCRIPTION
**Pull request by PythonCoderAS**
#### Summary

Removes extraneous `\`s messing up markup

#### Motivation

As a part of https://github.com/mdn/yari/issues/5074, I am trying to fix obvious formatting errors wherever possible.

#### Supporting details

N/A

#### Related issues

Fix #11259

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/accessibility/accessibility_and_spacial_patterns/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ NASA conducted research on the perception of color, and found that luminance con
...
 
-**\*"Spatial Localization.** Symbols which have the same luminance as their background are perceptually less securely located in space and time than are symbols with higher luminance contrast. They tend to "float" visually or be "captured" by adjacent symbols with high luminance-contrast. The phenomenon seems to be especially problematic for symbol/background combinations that differ only in the blue channel."\*
+***"Spatial Localization.** Symbols which have the same luminance as their background are perceptually less securely located in space and time than are symbols with higher luminance contrast. They tend to "float" visually or be "captured" by adjacent symbols with high luminance-contrast. The phenomenon seems to be especially problematic for symbol/background combinations that differ only in the blue channel."*
```
Comment (hamishwillee) [inline files/en-us/web/accessibility/accessibility_and_spacial_patterns/index.md:18]: If you look at the rendering this is not quite right - the first " is in bold.

I'd remove the " marks and the extra * and perhaps just render this as a block quote - ie put > in front of it

Comment (PythonCoderAS) [reply files/en-us/web/accessibility/accessibility_and_spacial_patterns/index.md:18]: I agree, it's that just I wanted to make the most obvious change first and then reformat as necessary. Is this better?

```suggestion
> Spatial Localization: Symbols which have the same luminance as their background are perceptually less securely located in space and time than are symbols with higher luminance contrast. They tend to "float" visually or be "captured" by adjacent symbols with high luminance-contrast. The phenomenon seems to be especially problematic for symbol/background combinations that differ only in the blue channel."
```

Comment (hamishwillee) [reply files/en-us/web/accessibility/accessibility_and_spacial_patterns/index.md:18]: Does to me. Let's see how it renders.

File: files/en-us/web/accessibility/accessibility_and_spacial_patterns/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ NASA conducted research on the perception of color, and found that luminance con
...
 
-**\*"Spatial Localization.** Symbols which have the same luminance as their background are perceptually less securely located in space and time than are symbols with higher luminance contrast. They tend to "float" visually or be "captured" by adjacent symbols with high luminance-contrast. The phenomenon seems to be especially problematic for symbol/background combinations that differ only in the blue channel."\*
+> Spatial Localization: Symbols which have the same luminance as their background are perceptually less securely located in space and time than are symbols with higher luminance contrast. They tend to "float" visually or be "captured" by adjacent symbols with high luminance-contrast. The phenomenon seems to be especially problematic for symbol/background combinations that differ only in the blue channel.
```
Comment (hamishwillee) [inline files/en-us/web/accessibility/accessibility_and_spacial_patterns/index.md:18]:
```suggestion
> **Spatial Localization:** Symbols which have the same luminance as their background are perceptually less securely located in space and time than are symbols with higher luminance contrast. They tend to "float" visually or be "captured" by adjacent symbols with high luminance-contrast. The phenomenon seems to be especially problematic for symbol/background combinations that differ only in the blue channel.
```

File: files/en-us/web/accessibility/accessibility_and_spacial_patterns/index.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ NASA conducted research on the perception of color, and found that luminance con
...
 
-**\*"Spatial Localization.** Symbols which have the same luminance as their background are perceptually less securely located in space and time than are symbols with higher luminance contrast. They tend to "float" visually or be "captured" by adjacent symbols with high luminance-contrast. The phenomenon seems to be especially problematic for symbol/background combinations that differ only in the blue channel."\*
+> **Spatial Localization:** Symbols which have the same luminance as their background are perceptually less securely located in space and time than are symbols with higher luminance contrast. They tend to "float" visually or be "captured" by adjacent symbols with high luminance-contrast. The phenomenon seems to be especially problematic for symbol/background combinations that differ only in the blue channel.
```
Comment (hamishwillee) [inline files/en-us/web/accessibility/accessibility_and_spacial_patterns/index.md:18]: Played around a bit. Reverted to almost the same as initially :-)

```suggestion
"_**Spatial Localization.** Symbols which have the same luminance as their background are perceptually less securely located in space and time than are symbols with higher luminance contrast. They tend to "float" visually or be "captured" by adjacent symbols with high luminance-contrast. The phenomenon seems to be especially problematic for symbol/background combinations that differ only in the blue channel._"
```

**Review by hamishwillee — APPROVED** [review]
I'm happy with this now. Thanks.

**Pull request merged**